conf/general.yml-example

```diff
@@ -28,6 +28,9 @@ SECURE_PROXY_SSL_HEADER: ''
 # If you're behind a proxy, set this to a two-element list containing the
 # trusted HTTP header and the required value. For example:
 #   SECURE_PROXY_SSL_HEADER: [ 'X-Forwarded-Proto', 'https' ]
+CONTENT_SECURITY_POLICY: ''
+# To activate CSP, set this to 1. If you have additional third party domains to
+# allow JavaScript on, set this to space-separated list of domains.
```

Comment:
It'd be nice to have this as a YAML list to be consistent with other settings e.g. `ALLOWED_COBRANDS`, `MAPIT_TYPES`, etc, unless there's another use case for being able to include arbitrary strings in the header?

Reply:
You could theoretically use it if you were happy using e.g. new CSP3 headers. But no reason it couldn't be a list then as well, they're all space separated items anyway. But I thought a string might make it clearer that it can be more than just domains, just I imagined that would be the common case. The content of this string inserts itself into the `script-src` part of the CSP. I'll try and update the help text.